controller/api/public/top_routes.go

```diff
@@ -13,6 +13,7 @@ import (
 
 const (
 	routeReqQuery             = "sum(increase(route_response_total%s[%s])) by (%s, dst, classification)"
+	actualRouteReqQuery       = "sum(increase(route_actual_response_total%s[%s])) by (%s, dst, classification)"
```

Comment:
at the prometheus metrics level and `BasicStats` level, we're re-using the existing response keys for "effective" stats, and adding new keys for "actual". higher up in the abstractions, around `jsonRouteStats` and cli table `headers`, we're instead adding both new "effective" and "actual" keys, and leaving the existing keys empty. what do you think about modifying the higher-level abstractions to match these lower-level ones, such that we'd remove "effective" altogether?